tests/tensor_search/integ_tests/test_no_model.py

```diff
@@ -137,18 +140,68 @@ def test_no_model_raise_error_if_query_in_search(self):
             with (self.subTest(index_name=index_name)):
                 with self.assertRaises(InvalidArgError) as e:
                     r = tensor_search.search(config=self.config, index_name=index_name, text="test")
-                self.assertIn("'no_model' cannot vectorise your content.", str(e.exception))
+                self.assertIn("Cannot vectorise anything with 'no_model'", str(e.exception))
 
     def test_no_model_work_with_context_vectors_in_search(self):
         """Test to ensure that context vectors work with no_model by setting query as None"""
+
+        custom_vector = [0.655 for _ in range(self.DIMENSION)]
+
+        docs = [
+            {
+                "_id": "1",
+                "custom_field_1":
+                    {
+                        "content": "test custom field content_1",
+                        "vector": np.random.rand(self.DIMENSION).tolist()
+                    }
+            },
+            {
+                "_id": "2",
+                "custom_field_1":
+                    {
+                        "content": "test custom field content_2",
+                        "vector": custom_vector
+                    }
+            }
+        ]
+
+        for index_name in [self.structured_index_with_no_model, self.unstructured_index_with_no_model]:
+            with (self.subTest(index_name=index_name)):
+                tensor_fields = ["custom_field_1"] if \
+                    index_name == self.unstructured_index_with_no_model else None
```

Comment:
An easier way to do this in future tests is: `if isinstance(index, UnstructuredMarqoIndex)`